build.xml

```diff
@@ -252,6 +253,15 @@
         <matches pattern="true" string="${build.isArmSoftFloat}"/>
       </and>
     </condition>
+    <condition property="jre.arch" value="mips64el">
+      <or>
+	<matches pattern="mips64el" string="${os.arch}"/>
+	<and>
+	  <matches pattern="mips64" string="${os.arch}"/>
+	  <matches pattern="little" string="${sun.cpu.endian}"/>
+	</and>
```

Comment:
I tested Zero and OpenJDK 8 which is ported by Loongson, they both report mips64el. Therefore, I agree that the second match (lines 259-262) should be removed.